.github/workflows/sql-test-and-build-workflow.yml

```diff
@@ -22,9 +22,6 @@ jobs:
     - name: Build with Gradle
       run: ./gradlew build assemble
 
-    - name: Run backward compatibility tests
-      run: ./scripts/bwctest.sh
```

Comment:
Your PR is showing commits from other work.  I'd suggest that you rebase your branch to remove those commits.